labs/coding-101-rest-basics-ga/7.md

```diff
@@ -14,21 +14,23 @@
 		* `C.` Enter the 'Content-Type' and value 'application/json'.
 		* `C.` Enter the 'X-Auth-Token' and the token created earlier. The 'X-Auth-Token' is required for all DNA Center API calls.
 	* **Authentication**
-		* Handled in the Headers
+		* Handled in the header.
 	* **Body**
-	 	* not required for this request
+	 	* Not required for this request.
 3. Click **Send**. Postman sends the request to the server.
 
 4. Review the response.
 	* The **Response Code** in the **Status** field should be `200 OK`.
 	* The **JSON** should display a list of hosts.
 
-    ![](/posts/files/coding-101-rest-basics-ga/assets/images/postman5.png "Response")
+    ![](assets/images/postman5.png "Response")
 
-## Things to Try
+## Things to try
 1. In Postman, append '?limit=1&offset=1' to the URL https://sandboxdnac/api/v1/host so reads as 'https://sandboxdnac/api/v1/host?limit=1&offset=1'. Click the **Send** button. How does the return data differ?  Check the API Reference Guide for more information about the Get Host API call.
 <br/><br/>
 
-#### Congratulations! You made your first REST API call using the APIC-EM Service Ticket!
+#### Congratulations! You made your first REST API call using the DNA Center Service Ticket!
```

Comment:
I think we call it a Token now instead of Ticket.

Reply:
I can make that change.